src/Tokens/TokensList/EthBalance/EthBalance.js

```diff
@@ -4,22 +4,23 @@
 // SPDX-License-Identifier: MIT
 
 import React, { Component } from 'react';
-import { balanceOf$, defaultAccount$ } from '@parity/light.js';
+import { myBalance$ } from '@parity/light.js';
 import { fromWei } from '@parity/api/lib/util/wei';
 import { Link } from 'react-router-dom';
-import { map, switchMap } from 'rxjs/operators';
+import { map } from 'rxjs/operators';
+import PropTypes from 'prop-types';
 
 import BalanceLayout from '../BalanceLayout';
-import light from '../../hoc';
+import light from '../../../hoc';
```

Comment:
True. At the end, since this hoc is really React bindings for the light.js library, it'll probably make more sense to create a separate repo, i.e. the oo7-react equivalent.